src/app/api/geolocation/route.ts

```diff
@@ -0,0 +1,46 @@
+import { NextResponse, type NextRequest } from "next/server"
+import { geolocation } from "@vercel/functions"
+
+// All EU/EEA country codes
+const GDPR_COUNTRIES = new Set([
+	"AT", // Austria
```

Comment:
can we replace tabs with spaces?

also the same comment as the console PR: the list doesn't cover all EU countries, a source would be great.